src/resources/views/crud/columns/select_multiple.blade.php

```diff
@@ -1,15 +1,22 @@
 {{-- relationships with pivot table (n-n) --}}
 @php
     $results = data_get($entry, $column['name']);
+
+    if(!$results->isEmpty()) {
+        $related_model_key = $results->first()->getKeyName();
+    }
 @endphp
 
 <span>
-    <?php
-        if ($results && $results->count()) {
-            $results_array = $results->pluck($column['attribute']);
-            echo implode(', ', $results_array->toArray());
-        } else {
-            echo '-';
-        }
-    ?>
-</span>
\ No newline at end of file
+
+        @if ($results && $results->count())
+            @php($results_array = $results->pluck($column['attribute'],$related_model_key)->toArray())
```

Comment:
Any chance we can move most of this logic above, in the PHP section, to leave the <span> cleaner like in other fields?